remoteit/desktop
# [DESK-908] read config cli overrides

### Description

read config CLI overrides

### Checklist

### Test plan

1. Add settings [apiURL, betaApiURL] in your `config.json` for `overrides`,
2. Sign in
3. Check network request
4. You should see requests using your own setting from config.json

### Ticket

https://remoteit.atlassian.net/browse/DESK-908
